code/modules/clothing/suits/medieval_armor.dm

```diff
@@ -26,6 +26,26 @@
 	icon_state = "knight_orange"
 	item_state = "knight_orange"
 
+/obj/item/clothing/head/helmet/medieval/alt
+	icon_state = "knight_helmet"
+	item_state = "knight_helmet"
+
+/obj/item/clothing/head/helmet/medieval/paladin
+	name = "elite paladin helm"
+	desc = "This tarnished helmet has clearly done its job well for many years of service."
+	icon_state = "paladin"
+	item_state = "paladin"
+	action_button_name = "Toggle Visor"
+
+/obj/item/clothing/head/helmet/medieval/paladin/attack_self(mob/user as mob)
+	if(src.icon_state == initial(icon_state))
+		src.icon_state = "[icon_state]1"
+		to_chat(user, "You raise the helmet's visor.")
+	else
+		src.icon_state = initial(icon_state)
+		to_chat(user, "You lower the helmet's visor.")
+	update_worn_icon()	//so our mob-overlays update"
```

Comment:
this could also toggle between face coverage and not, but thats just a minor thing